functional_tests/.gitignore

```diff
@@ -0,0 +1 @@
+!cloudbuild.yaml
```

Comment:
The .gitignore file at root excludes cloudbuild.yaml files. For this, we do want to include cloudbuild.yaml files, hence the rule.